core/src/impl/Kokkos_Profiling_C_Interface.h

```diff
@@ -28,7 +28,7 @@
 #include <stdbool.h>
 #endif
 
-#define KOKKOSP_INTERFACE_VERSION 20211015
+#define KOKKOSP_INTERFACE_VERSION 20240906
```

Comment:
Do we have a way to (or do we care to) document what has changed when this interface version changes?

Reply:
I don't think we currently have a way to do it.
Here the change would capture that fallback kernel names are not generated using the `typeid` operator any more and that tool could in principle skip demangling.